taichi/codegen/spirv/spirv_codegen.cpp

```diff
@@ -2729,7 +2735,7 @@ void KernelCodegen::run(TaichiKernelAttributes &kernel_attribs,
     bool success = true;
     {
       bool result = false;
-      TI_ERROR_IF(
+      TI_WARN_IF(
```

Comment:
nit: shall we keep this error to prevent accidental perf regression? If a PR breaks the spriv-opt we'd like to know that asap I guess.

Reply:
I think there are cases spv-opt will fail, and this message will come out but the kernel is actually fine. I think the later pipeline will catch if the pipeline is truly broken